docker/k8s/README.md

```diff
@@ -7,14 +7,18 @@ in SGX-SIM mode.
 ## Prerequisites
 - kubectl-v1.15.2
 - a kubernetes cluster
+- To enable SGX HW, follow the instructions in
+  [PREREQUISITES.md](../../PREREQUISITES.md#intel-sgx-in-hardware-mode)
+- To use SGX HW, build docker image using below command  
+  `docker-compose -f docker-compose-sgx.yaml build`
 
 ## Running 
 
 1. Initialize the common configuration as a ConfigMap
     ```bash
     kubectl create -f tcf-config-map.yaml
     ```
-2. Enable `remote_url` in [tcs_config.toml](../../config/tcs_config.toml)
+2. Enable `remote_url` in [tcs_config.toml](../../config/tcs_config.toml)  
```

Comment:
Remove the trailing whitespace you added to this line.

Reply:
two trailing spaces is new line in "mark down" document syntax.